src/containers/BuySellCrypto/PaymentMethod/SendTransaction/SendTransaction.js

```diff
@@ -0,0 +1,145 @@
+                                :
+                                `For ${this.state.toVal} "${this.state.toCurr}"`
+                            }
+                        </Text>
+                    </View>    
+                    <View style={styles.buttonContainer}>
+                        <Button1
+                            style={styles.saveChangesButton}
+                            buttonContent="Confirm"
+                            onPress={this.handleConfirm}
+                        />
+                        <Button1
+                            style={styles.backButton}
+                            buttonContent="Cancel"
+                            onPress={this.back}
+                        />
+                    </View>
+                </View>
+                <Text style={styles.textTransactionHistory}>Transaction history:</Text>
+                <ScrollView>
+                    <View style={styles.containerTransactionHistory} >
+                        {
+                            this.props.history && this.props.history.map((item) => (
+                                <View key={item.id}>
+                                    <ListItem
+                                        title={`${item.sourceCurrency==='USD' ? 'Bought' : 'Sold' } ${item.sourceAmount} ${item.sourceCurrency}`}
+                                        titleStyle={styles.itemTextTransactionHistory}
+                                        subtitle={`For ${item.destAmount} ${item.destCurrency}`}
+                                        subtitleStyle={styles.itemTextTransactionHistory}
+                                        rightTitle={new Date(item.createdAt).toISOString().split('.')[0]}
+                                        rightTitleStyle={styles.itemTextTransactionHistory}
+                                        hideChevron={true}
+                                        containerStyle={styles.itemContainerTransactionHistory}
+                                    />
+                                    <ListItem 
+                                        title={item.id} 
+                                        titleStyle={styles.itemIdTransactionHistory}
+                                        hideChevron={true}
+                                        rightTitle={item.status}
+                                        rightTitleContainerStyle={styles.rightTitleContainer}
+                                        rightTitleStyle={styles.rightTitleStyle}
+                                    />
+                                </View>
+                            ))
+                            
+                        }
+                    </View>
+                </ScrollView>
+            </View>
+        );
+    }
+}
+
+const mapStateToProps = (state) => ({
+    activeAccount: state.authentication.activeAccount,
+    isTransactionHistoryFetching: selectTransactionHistoryIsFetching(state),
```

Comment:
transactionHistoryIsFetching

Reply:
You are not using this prop at the moment. If you have it, add a loader to the component while it's fetching the data